Gem does not compile on Centos7 using Ruby 2.7.0

I'm unable to compile seven_zip_ruby on ruby 2.7.0.  It works fine on 2.5.x and 2.6.x on the same machine.  Also works on 2.7.0 on mac.  I haven't tried any other linux distributions.

Relevant failure message:
```
seven_zip_archive.cpp: In lambda function:
seven_zip_archive.cpp:1539:13: error: ‘ruby_nonempty_memcpy’ is not a member of ‘std’
             std::memcpy(data, RSTRING_PTR(str), RSTRING_LEN(str));
             ^
seven_zip_archive.cpp:1539:13: note: suggested alternative:
In file included from /var/jenkins/.rvm/rubies/ruby-2.7.0/include/ruby-2.7.0/ruby.h:33:0,
                 from seven_zip_archive.h:27,
                 from seven_zip_archive.cpp:10:
/var/jenkins/.rvm/rubies/ruby-2.7.0/include/ruby-2.7.0/ruby/ruby.h:1758:1: note:   ‘ruby_nonempty_memcpy’
 ruby_nonempty_memcpy(void *dest, const void *src, size_t n)
 ^
make: *** [seven_zip_archive.o] Error 1

make failed, exit code 2
```

--- a/ext/seven_zip_ruby/seven_zip_archive.cpp
+++ b/ext/seven_zip_ruby/seven_zip_archive.cpp
@@ -1536,7 +1536,7 @@ STDMETHODIMP InStream::Read(void *data, UInt32 size, UInt32 *processedSize)
     bool ret = m_archive->runRubyAction([&](){
         VALUE str = rb_funcall(m_stream, INTERN("read"), 1, ULONG2NUM(size));
         if (!NIL_P(str) && data){
-            std::memcpy(data, RSTRING_PTR(str), RSTRING_LEN(str));
+            memcpy(data, RSTRING_PTR(str), RSTRING_LEN(str));
         }
 
         if (processedSize){

